frontend/src/metabase/reference/databases/TableSidebar.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint "react/prop-types": "warn" */
+import React from "react";
+import PropTypes from "prop-types";
+import S from "metabase/components/Sidebar.css";
+
+import Breadcrumbs from "metabase/components/Breadcrumbs.jsx";
+import SidebarItem from "metabase/components/SidebarItem.jsx"
+
+import cx from 'classnames';
+import pure from "recompose/pure";
+
+const TableSidebar = ({
+    database,
+    table,
+    style,
+    className
+}) =>
+    <div className={cx(S.sidebar, className)} style={style}>
+        <ul>
```

Comment:
Bit pedantic but I'd change this from an `<ul>` to an `<ol>` since the order of the items in the sidebar is significant. I'd also scoot that down to wrap the side bar items since a list should have list items as its immediate children.